Meta/check-newlines-at-eof.py

```diff
@@ -0,0 +1,73 @@
+#!/bin/python3
+
+import os
+import subprocess
+import sys
+
+try:
+    os.chdir(os.path.dirname(__file__) + "/..")
+except:
+    os.exit(1)
```

Comment:
Raw `except` is almost never what you want, something like `except OSError` is more appropriate. I don't think the try/except is necessary at all though, just let it raise, that will exit the script as well (and shows the error to the user rather than exiting silently).

Reply:
you are right.